src/index.ts

```diff
@@ -9,6 +9,7 @@ export { Rule as DirectiveSelectorRule } from './directiveSelectorRule';
 export { Rule as I18nRule } from './i18nRule';
 export { Rule as TemplateCyclomaticComplexityRule } from './templateCyclomaticComplexityRule';
 export { Rule as TemplateConditionalComplexityRule } from './templateConditionalComplexityRule';
+export { Rule as InlineTemplateMaxLinesRule } from './inlineTemplateMaxLinesRule';
```

Comment:
Since we offer a check for css and template, we should rename the rule like this: `MaxInlineDeclarationsRule` (and the file name)

Reply:
Oh, thanks for catching this! Fixed